packages/service-library/src/servicelib/rabbitmq.py

```diff
@@ -184,8 +184,8 @@ async def subscribe(
 
         assert self._channel_pool  # nosec
         async with self._channel_pool.acquire() as channel:
-            _DEFAULT_PREFETCH_VALUE = 10  # this value is set to the default for now
-            await channel.set_qos(_DEFAULT_PREFETCH_VALUE)
+            _PREFETCH_VALUE = 1 if exclusive_queue is False else 10
```

Comment:
Please set the 10 as an external constant, such as _DEFAULT_PREFETCH_VALUE. we might want to adjust that in the future

Reply:
done :+1: